android/src/main/java/com/rnldk/RnLdkModule.kt

```diff
@@ -142,6 +142,13 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
           that.sendEvent(MARKER_PERSIST_MANAGER, params);
         }
       }
+
+      override fun persist_network_graph(network_graph: ByteArray?) {
```

Comment:
@marcosrdz can you pls check swift part and implement this event handler as well.
@arik-so its weird that it was not forced on swift

Reply:
ok, I'll update the build to make it forced

Reply:
try this: https://github.com/lightningdevkit/ldk-swift/releases/tag/v0.0.106.1